packages/charts-react-line/src/LineChart.story.js

```diff
@@ -32,5 +32,15 @@ storiesOf('LineChart', module)
       domainY={extent(data, (d) => d.close)}
       x={(d) => d.date}
       y={(d) => d.close}
+      grid={[
+        {
+          text: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
```

Comment:
I have not connected the flow from data to the grid prop yet. I was planning for that to be the next step!